scripts/migration/migrate_registration_extra.py

```diff
@@ -0,0 +1,53 @@
+"""
+Changes existing question.extra on a draft to a list
+required for multiple files attached to a question
+"""
+import sys
+import logging
+
+from website.app import init_app
+from scripts import utils as scripts_utils
+from website.models import DraftRegistration
+from framework.transactions.context import TokuTransaction
+
+
+
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.INFO)
+
+def migrate_drafts(dry):
+    draft_registrations = DraftRegistration.find()
```

Comment:
Is there a reason not to filter this set ahead of time? Like:
https://github.com/CenterForOpenScience/osf.io/blob/847824cf86ddd7403a9b18b1ff3278cd8399ab63/website/prereg/views.py#L26